tasks/24_index-to-insert/index-to-insert.js

```diff
@@ -0,0 +1,7 @@
+export const getIndexToInsert = (arr, arg) => {
+  let sortArr = arr.sort((a, b) => a - b);
+  let indexNum = sortArr.map((cur) => cur >= arg ? true : false).indexOf(true);
```

Comment:
don't need to return ? true: false in arrow function. statement cur >= arg is enough.

Why do you need indexof(true)?